spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -20,11 +42,11 @@ public interface CarRepository extends BasicRepository<Car, Long> {
 }
 ----
 
-In the above example, we have a `CarRepository` interface that extends the `BasicRepository` interface provided by Jakarta EE Data. The `BasicRepository` interface offers a set of basic operations for entities.
+In the above example, a `CarRepository` interface extends the `BasicRepository` interface provided by Jakarta EE Data. The `BasicRepository` interface offers a set of basic operations for entities.
 
-By annotating the `CarRepository` interface with `@Repository`, we indicate that it is a repository component managed by the Jakarta EE Data framework. It allows the Jakarta Data provider to automatically generate the necessary implementations for the defined methods, saving us from writing boilerplate code.
+By annotating the `CarRepository` interface with `@Repository`, the application requests that the Jakarta Data provider generate implementation of the interface and its methods. To make the implementation available to the application, the Jakarta Data provider uses CDI to register the implementation as a bean or registers a producer or producer factory to provide the implementation on unqualified injection points for the `CarRepository` interface.
```

Comment:
```suggestion
By annotating the `CarRepository` interface with `@Repository`, the application requests that the Jakarta Data provider generate an implementation of the interface and its methods. To make the implementation available to the application, the Jakarta Data provider uses CDI to register the implementation as a bean, or registers a producer or producer factory to provide the implementation on unqualified injection points for the `CarRepository` interface.
```

Reply:
The first part of the suggested change (inserting "an") looks correct, but the second part to add a comma looks incorrect because it is between two verbs ("uses" and "registers") for the same subject ("the Jakarta Data provider").

It would be correct to add the comma if we also add the word "it" so that it becomes a compound sentence,
> ... the Jakarta Data provider uses CDI to register the implementation as a bean, or it registers ...

Reply:
Yep, you're right that's a compound predicate which would be wrong to have a comma, I'll update it.

Reply:
Thanks, I'll add it now.